mailer.rst

```diff
@@ -1437,6 +1437,65 @@ is sent::
         }
     }
 
+SentMessageEvent
+~~~~~~~~~~~~
+
+``SentMessageEvent`` it allows acting on the :class:`Symfony\\Component\\Mailer\\SentMessage`::
```

Comment:
Perhaps document like this?
https://symfony.com/doc/current/reference/events.html#kernel-request

When and why and what can be done in it

Reply:
Good idea !
Personally I like it when it's uniform (it's simpler and nicer to read). So I standardized like https://symfony.com/doc/current/reference/events.html#kernel-request 

I have detailed this event. Thanks for your review